Sources/SwiftDocCUtilities/Action/Actions/Convert/JSONEncodingRenderNodeWriter.swift

```diff
@@ -115,6 +115,7 @@ class JSONEncodingRenderNodeWriter {
             withIntermediateDirectories: true,
             attributes: nil
         )
-        try fileManager.createFile(at: htmlTargetFileURL, contents: indexHTML)
+        
+        try fileManager.copyItem(at: indexHTML, to: htmlTargetFileURL)
```

Comment:
Yeah I was surprised by this- I think it's still the better design though since it allows the OS to potentially reduce size-on-disk by using copy-on-write behavior.